app/src/main/java/com/bogdan/codeforceswatcher/features/users/redux/requests/UsersRequests.kt

```diff
@@ -12,10 +12,13 @@ import com.bogdan.codeforceswatcher.room.DatabaseClient
 import com.bogdan.codeforceswatcher.store
 import org.rekotlin.Action
 
+enum class Source { USER, BROADCAST }
+
 class UsersRequests {
 
     class FetchUsers(
-        private val isInitiatedByUser: Boolean
+        private val isToastNeeded: Boolean,
```

Comment:
I would say that `source` can be `BACKGROUND, USER, BROADCAST` and then you decide if it's needed to show a Toast (only for `source -> USER`).